pkg/audit/verifier.go

```diff
@@ -38,50 +38,33 @@ type Share struct {
 
 // Verifier helps verify the correctness of a given stripe
 type Verifier struct {
-	orders  *orders.Service
-	auditor *identity.PeerIdentity
-
-	downloader downloader
-}
-
-type downloader interface {
-	DownloadShares(ctx context.Context, limits []*pb.AddressedOrderLimit, stripeIndex int64, shareSize int32) (shares map[int]Share, nodes map[int]storj.NodeID, err error)
-}
-
-// defaultDownloader downloads shares from networked storage nodes
-type defaultDownloader struct {
-	log       *zap.Logger
-	transport transport.Client
-	overlay   *overlay.Cache
-	reporter
-
+	log               *zap.Logger
+	orders            *orders.Service
+	auditor           *identity.PeerIdentity
+	transport         transport.Client
+	overlay           *overlay.Cache
 	minBytesPerSecond memory.Size
 }
 
-// newDefaultDownloader creates a defaultDownloader
-func newDefaultDownloader(log *zap.Logger, transport transport.Client, overlay *overlay.Cache, id *identity.FullIdentity, minBytesPerSecond memory.Size) *defaultDownloader {
-	return &defaultDownloader{log: log, transport: transport, overlay: overlay, minBytesPerSecond: minBytesPerSecond}
-}
-
 // NewVerifier creates a Verifier
 func NewVerifier(log *zap.Logger, transport transport.Client, overlay *overlay.Cache, orders *orders.Service, id *identity.FullIdentity, minBytesPerSecond memory.Size) *Verifier {
-	return &Verifier{downloader: newDefaultDownloader(log, transport, overlay, id, minBytesPerSecond), orders: orders, auditor: id.PeerIdentity()}
+	return &Verifier{log: log, orders: orders, auditor: id.PeerIdentity(), transport: transport, overlay: overlay, minBytesPerSecond: minBytesPerSecond}
 }
 
 // Verify downloads shares then verifies the data correctness at the given stripe
-func (verifier *Verifier) Verify(ctx context.Context, stripe *Stripe) (verifiedNodes *RecordAuditsInfo, err error) {
+func (v *Verifier) Verify(ctx context.Context, stripe *Stripe) (verifiedNodes *RecordAuditsInfo, err error) {
```

Comment:
We have to delete such convention https://github.com/storj/docs/blob/master/code/Style.md#variable-naming

Otherwise it is misguiding

Reply:
oh, thanks! I heard you mention this in relation to https://github.com/golang/go/wiki/CodeReviewComments#receiver-names.  I misunderstood our preference.

Reply:
you're not he only one using short names, so I was wondering if the people didn't like to use shorter ones or basically they aren't aware of it.

I'll comment to the next reviews if I see them, because it looks that it's more about people aren't aware.